.github/workflows/autobuild.yaml

```diff
@@ -128,11 +227,22 @@ jobs:
         ninja -C grpc-build install
     - name: Fetch target commit
       uses: actions/download-artifact@v1
-      with: { name: release }
-    - name: Set target commit
+      with:
+        name:
+          env
+    - name: Fetch release info
+      uses: actions/download-artifact@v1
+      with:
+        name:
+          release
+    - name: Get release info
+      run: |
+        cat env/commit.env >> $GITHUB_ENV
+        cat release/release.env >> $GITHUB_ENV
+      shell: bash
```

Comment:
we didn't need this in previous steps? can you put a comment explaining why it is needed here?

Reply:
Sure! As you can see, before we used `${{ github.event.release.tag_name }}` which is "global". Now, we are using env variables we are putting to `$GITHUB_ENV` and they're not in the context before the step finishes, so this now needs to be a separate step.